### ISSUE TITLE
"Reverse Engineer" and most other options are missing in context menu

### ISSUE THREAD

**Issue by AndreErb** [ErikEJ/EFCorePowerTools#2188]
### Provide steps to reproduce a bug
1. Select Project in VS 2022
2. Right-Click
3. No "Reverse Engineer" option is shown

The "Reverse Engineer" and most other options are missing. Only some options are shown.
![image](https://github.com/ErikEJ/EFCorePowerTools/assets/11599017/4495b834-536d-44f4-9ef2-ee03d6d7b25d)

I tried uninstalling and re-installing the extension, but the issue remains.

### Provide technical details

- EF Core Power Tools version: **2.6.140**

- Exact Visual Studio version: Visual Studio 2022 **17.8.7**

- Database engine: SQL Server

- EF Core version in use: Doesn't matter

**Comment by ErikEJ** [comment]
Please share your. Csproj - I limit options based on project type

**Comment by AndreErb** [comment]
> Please share your. Csproj - I limit options based on project type

Oh I see

I had `<TargetFramework>netcoreapp3.1</TargetFramework>` in all the projects I tried/tested with.

Changing it and moving on to .NET8 with `<TargetFramework>net8.0</TargetFramework>` fixed it and brought back the options. Didn't notice, that support for older targets was removed.

Thank you for pointing me into the right direction.
Will close this.
